portal/src/main/scala/cromwell/pipeline/datastorage/dao/entry/ProjectEntry.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.pipeline.datastorage.dao.entry
+
+import cromwell.pipeline.datastorage.Profile
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectId, UserId }
+
+trait ProjectEntry extends Profile {
+  self: UserEntry =>
```

Comment:
Looks like sleftypes are the only way indeed.
Just a small sugar request: make it `trait ProjectEntry { this: Profile with UserEntry =>`.

And the same thing with `UserEntry`

Reply:
updated both entries